design/sd-zip-artifacts.md

```diff
@@ -134,12 +134,21 @@ This is a new component.
 1. If retry fails
     1. Add a statusMessage to the build to notify the user that the unzip has failed
     1. Log the failure.
+1. Delete SD_ARTIFACT.zip
 
 ### SD Store (store)
+Enable the following operations from unzip worker
+1. Download SD_ARTIFACT.zip
+1. Upload unzipped artifact files
+1. Delete SD_ARTIFACT.zip
 
-1. Upload and Download artifact files
+#### API
 
-SD Store do not need to add new function, but need to add new Authentication.
+|Method|URL|Description|
+|:--|:--|:--|
+|DELETE|/builds/{id}/ARTIFACTS/SD_ARTIFACT.zip|delete zipped artifact files|
```

Comment:
I feel a little bit strange the path of `SD_ARTIFACT.zip`.
Is it better to path filename with body or parameter?

Reply:
I'm using the same implementation as `DELETE /caches` and `DELETE /commands`.
- https://github.com/screwdriver-cd/store/blob/176fc403741f462d95a2f0bfe1dae26fd724f5a1/plugins/caches.js#L303-L434
- https://github.com/screwdriver-cd/store/blob/176fc403741f462d95a2f0bfe1dae26fd724f5a1/plugins/commands.js#L165-L212

Reply:
When I implement this API literally the path will be `path: '/builds/{id}/SD_ARTIFACT.zip` not like `path: '/builds/{id}/{filename}`. 

There is a possibility that the zip file name will change,  at that time the path of `SD_ARTIFACT.zip` is strange.

Reply:
I fixed the path.